lib/index.js

```diff
@@ -69,7 +69,7 @@ const send = (res, code, obj = null) => {
     }
 
     if (!res.getHeader('Content-Type')) {
-      res.setHeader('Content-Type', 'application/json')
+      res.setHeader('Content-Type', 'application/json; encoding=utf-8')
```

Comment:
I think this should be `charset=utf-8` instead of `encoding=utf-8` by [RFC2068](https://www.ietf.org/rfc/rfc2068.txt) and [the official w3 article](https://www.w3.org/International/articles/http-charset/index).

Reply:
Oh, right! Thanks!